README.md

```diff
@@ -34,6 +33,47 @@ Special notes:
  * Since we cannot distribute Java Richards, a patch is required to derive the
    tracing version (`patches/trace_java_richards.diff`)
 
+## Using the scripts here to analyse an existing benchmark
+
+### Mandatory requirements
+
+  * Python 2.7 (the code here is not Python 3.x ready)
+  * bzip2 / bunzip2
+  * Python modules: numpy, rpy2
+  * R
+
+### Optional requirements
+
+  * Python modules required for plotting: matplotlib, seaborn
+  * Required for generating LaTeX tables: pdflatex
+
+### CSV format
+
+The scripts here take CSV files as input. The format is as follows. The first row
+must contain a header with a process execution id, benchmark name and sequence
+of iteration numbers. Subsequent rows should contain an id, benchmark name and
```

Comment:
`Subsequent rows are data rows, one per process execution. The in-process iteration index columns should contain the time in seconds for the corresponding in-process iteration` is probably enough?